### PR TITLE
Django 1.11 support

### PR DESCRIPTION
**Pull request by phala**

### CODE REVIEW

**Comment by phala** [comment]
@hedleyroos please review

File: object_tools/tests/test_object_tools_config.py [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+        # should be 'object-tools'.
+        self.assertEqual(tools.urls, ([], 'object-tools', 'object-tools'))
+
+        # With a tool registered, urls should include it for each model.
+        tools.register(TestTool)
+        urls = tools.urls
+        self.assertEqual(len(urls[0]), 6)
+        for url in urls[0]:
+            self.assertTrue(url.url_patterns[0].__repr__() in [
+                '<RegexURLPattern sessions_session_test_tool ^test_tool/$>',
```
Comment (hedleyroos) [inline object_tools/tests/test_object_tools_config.py]: It looks a bit odd to compare on an object's repr but let's leave it like this.

File: README.rst [hunk 2]
```diff
@@ -14,7 +14,7 @@ This packages is part of the larger `Jmbo <http://www.jmbo.org>`_ project.
 
 Requirements
 ------------
-* Django=>1.8
+* Django=>1.9
```
Comment (hedleyroos) [inline README.rst]: Remember to update the changelog indicating version support changes.

File: tox.ini [hunk 3]
```diff
@@ -1,27 +1,21 @@
...
 
-[testenv:django15-py27]
-commands = {envpython} manage.py test object_tools --settings=test_settings
+[testenv:django19-py27]
+commands =
+	{envpython} manage.py test --settings=object_tools.tests.settings.19
 
-[testenv:django14-py27]
-commands = {envpython} manage.py test object_tools --settings=test_settings
+[testenv:django19-py3]
```
Comment (hedleyroos) [inline tox.ini]: How does this tox test 110 and 111?

Comment (phala) [inline tox.ini]: `testenv` is the default that runs for 110 and 111.
Although, I'll add separate settings files for 110 and 111 and I'll then have to name the `testenv`s explicitly

Comment (phala) [inline tox.ini]: `testenv` is the default that runs for 110 and 111.
Although, I'll add separate settings files for 110 and 111 and I'll then have to name the `testenv`s explicitly

**Pull request merged**